newlib/libc/include/machine/setjmp.h

```diff
@@ -22,7 +22,15 @@ _BEGIN_STD_C
 #endif
 
 #if defined(__aarch64__)
-#define _JBLEN 22
+# if defined(__CYGWIN__)
+/* 
+ * Windows Arm64 ABI requires saving x19-x28, FP, LR, SP, FPCR, FPSR, q8-q15
+ * and jump address to jmp_buf.
+ */
+#  define _JBLEN 24
```

Comment:
Please take for the consideration that q8-q15 is 128 bits each

Reply:
Thanks for noticing, `d8-d15` registers should be saved.

Reply:
Comment fixed.